liblouis/compileTranslationTable.c

```diff
@@ -3820,11 +3821,15 @@ compileRule (FileInfo * nested)
 		case CTO_Match:
 		{
 			CharsString ptn_before, ptn_after;
-			widechar patterns[27720];
 			TranslationTableOffset offset;
 			int len, mrk;
 
-				memset(patterns, 0xffff, sizeof(patterns));
+			size_t patternsByteSize = sizeof(*patterns) * 27720;
+			patterns = (widechar*) malloc(patternsByteSize);
+			if(!patterns)
+				outOfMemory();
+			memset(patterns, 0xffff, patternsByteSize);
```

Comment:
I noticed that `malloc` is also used in many other places (25) of this same file (liblouis/compileTranslationTable.c) while `calloc` was not. Also `calloc` would set the value for these positions to 0 instead of 0xffff, so using `malloc` is more efficient if we will end up having to use `memset` anyway.